tests/suites/test_suite_pk.function

```diff
@@ -69,6 +69,322 @@ size_t mbedtls_rsa_key_len_func( void *ctx )
  * END_DEPENDENCIES
  */
 
+/* BEGIN_CASE */
+void valid_parameters( )
+{
+    mbedtls_pk_context pk;
+    size_t len;
+
+    mbedtls_pk_init( &pk );
+
+    TEST_VALID_PARAM( mbedtls_pk_setup( &pk, NULL ) ==
+                      MBEDTLS_ERR_PK_BAD_INPUT_DATA );
+
+    TEST_VALID_PARAM( mbedtls_pk_get_bitlen( NULL ) == 0 );
```

Comment:
As mentioned, I think it should be forbidden to pass `NULL` to `mbedtls_pk_can_do()`, as for the rest of the PK API. While parameter validation isn't suited to test this because it doesn't return `int`, and so we might not want to change the code for `mbedtls_pk_get_bitlen()`, at least we shouldn't add a test here suggesting that passing `NULL` is ok.

Reply:
Same as `can_do`

Reply:
Same as `can_do`